randovania/games/dread/json_data/Artaria.txt

```diff
@@ -2251,6 +2251,7 @@ Extra - asset_id: collision_camera_023_B
                   Spider Magnet or Space Jump
                   Speed Booster and Movement (Beginner) and Disabled Door Lock Randomizer and Open Charge Door
                   Grapple Beam and Grapple Movement (Intermediate)
+                  Flash Shift and Flash Shift Upgrade ≥ 3 and Combat (Beginner) and Movement (Intermediate)
```

Comment:
Just curious: Why we need movement and combat trick here?
Movement because of the...uh, flame thrower(???) things from above and combat because of the Plys?

Reply:
Going through and reviewing myself now. Movement would be the same as why it is needed for the above, plus having to actually charge the speed in that little space you have if door rando blocks you from doing so otherwise. Combat I'm not too sure about though why is there.

Reply:
I put Combat in because you have to shoot the Plys while you're Flash Shifting, and also avoiding the flames. They respawn quite quickly when leaving the room and they're usually too far left to be hit before you start flashing over, so it's often necessary to weave shooting in with the flash shifts ("chaining") to get through safely.

Reply:
oh this is flash shift, not speed. In that case, I would agree.

On that note, could probably add Speed Upgrade logic like i had thought it was.

Reply:
Looks like there's already a connection for Speed Booster, and I'm not sure having a Speed Upgrade really gains you much. Maybe if you can't open the door to the other hot room?